<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(31, 40, 51);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(197, 198, 199);
background-color: rgb(69, 162, 158);
font: 75 9pt &quot;Terminal&quot;;</string>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>493</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="verticalSpacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>100</number>
     </property>
     <property name="rightMargin">
      <number>100</number>
     </property>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color: rgb(69, 162, 158);
font: 75 12pt &quot;Terminal&quot;;</string>
       </property>
       <property name="text">
        <string>Keyboard trainer</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLineEdit" name="lineEdit">
       <property name="styleSheet">
        <string notr="true">color: rgb(197, 198, 199);
font: 75 10pt &quot;Trebuchet MS&quot;;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="styleSheet">
        <string notr="true">color: rgb(197, 198, 199);
font: 75 10pt &quot;Trebuchet MS&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">color: rgb(197, 198, 199);
background-color: rgb(69, 162, 158);
font: 75 9pt &quot;Terminal&quot;;</string>
       </property>
       <property name="text">
        <string>log in</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_2">
       <property name="styleSheet">
        <string notr="true">color: rgb(197, 198, 199);
background-color: rgb(69, 162, 158);
font: 75 9pt &quot;Terminal&quot;;</string>
       </property>
       <property name="text">
        <string>sign up</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>10</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(186, 0, 0);
font: 75 8pt &quot;Trebuchet MS&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">color: rgb(197, 198, 199);
font: 75 8pt &quot;Trebuchet MS&quot;;
</string>
       </property>
       <property name="text">
        <string>Введите ник</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">color: rgb(197, 198, 199);
font: 75 8pt &quot;Trebuchet MS&quot;;
</string>
       </property>
       <property name="text">
        <string>Введите пароль</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>527</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
